client/starwhale/core/eval/executor.py

```diff
@@ -212,10 +219,6 @@ def _gen_run_container_cmd(self, typ: str, step: str, task_index: int) -> str:
             "-v",
             f"{self.sw_config.object_store_dir}:{self.sw_config.object_store_dir}",
             "-v",
-            f"{self._model_dir}:{_CNTR_WORKDIR}/{RunSubDirType.SWMP}/src",
```

Comment:
Why remove the model dir mount?

Reply:
swmp and swrt has been removed, use artifact but not dir